.codespellrc

```diff
@@ -0,0 +1,6 @@
+# Lint Codespell configurations
+[codespell]
+skip = .codespellrc,.git,node_modules,build,dist,*.zst,CHANGELOG.md
+ignore-words-list = NotIn,AKS
+enable-colors =
+count =
```

Comment:
nit: the count was confusing me in the output - we can leave if you think it's valuable but my vote would be to drop this line.